script/setup.js

```diff
@@ -497,30 +499,38 @@ try {
 		successSpinnerBlock(`Finished API hydration.`);
 	}
 
-	await withSpinner(
-		async () => {
-			await fs.rm("./script", { force: true, recursive: true });
-			await fs.rm(".github/workflows/setup.yml");
-		},
-		{
-			startText: `Removing setup script...`,
-			successText: `Removed setup script.`,
-			stopText: `Error removing setup script.`,
-			errorText: `Could not remove setup script. `,
-		}
-	);
+	if (skipUninstalls) {
+		console.log();
+		console.log(chalk.gray`➖ Skipping removal of setup script.`);
```

Comment:
[Bug] This isn't skipped though? Why include the message?

```suggestion
		console.log(chalk.gray`➖ Skipping removal of setup script.`);
```

Reply:
I meant to log 'skipping removal of ```./script```.' Assuming that you want to keep that folder if the purpose of ```--skip-uninstalls``` is to work on the setup scripts.